src/mocks/server_request.ts

```diff
@@ -29,7 +43,7 @@ export const MockServerRequest = function (
   //
   //   TypeError: Cannot read property 'write' of undefined
   //
-  request.respond = function respond(output: any) {
+  request.respond = function respond(output: any) { // TODO RequestRespondOutput & different method signature than original
     output.send = function () {
```

Comment:
we can trash this. this code was taken from drash

Reply:
what part? the whole file, and its test file too?

Reply:
we can trash this `respond` method. we're going to deprecate `ServerRequest`, so it can stay a bit.

Reply:
Removed `request.respond` overwriting